tests/test_shed_tools_methods.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+
+from ephemeris.shed_tools_methods import *
```

Comment:
I think we should avoid `*` imports as a rule, makes it easier to follow what's being tested.